pytest_testrail/plugin.py

```diff
@@ -181,6 +183,10 @@ def pytest_sessionfinish(self, session, exitstatus):
                     self.add_results(testrun_id)
             else:
                 print('[{}] No data published'.format(TESTRAIL_PREFIX))
+            
+            if self.close_on_complete:
```

Comment:
Why only the last testrun is closed? What behavior is excepted when '`--tr-plan-id`' option is used?